packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -376,17 +384,16 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
                 onClick={ this._onAutofillClick }
                 onTouchStart={ this._onTouchStart }
                 onInputValueChange={ this._onInputChange }
-                aria-expanded={ isOpen }
                 aria-autocomplete={ this._getAriaAutoCompleteValue() }
-                role='combobox'
+                role='textbox'
                 aria-readonly={ ((allowFreeform || disabled) ? undefined : 'true') }
                 readOnly={ disabled || !allowFreeform }
                 aria-labelledby={ (label && (id + '-label')) }
                 aria-label={ ((ariaLabel && !label) ? ariaLabel : undefined) }
                 aria-describedby={ describedBy + (keytipAttributes['aria-describedby'] || '') }
                 aria-activedescendant={ this._getAriaActiveDescentValue() }
                 aria-disabled={ disabled }
-                aria-owns={ (id + '-list') }
+                aria-controls={ isOpen ? (id + '-list') : undefined }
```

Comment:
All of these changes are new patterns in aria 1.1. And the spec states that aria 1.0 patterns should still continue to be supported. We should verify that this reads out correctly in latest Edge and Narrator (at least) before making this change. I feel like this is making the code work for the widest array of browser/screen reader combos and we should only make this change once this pattern is widely supported